drivers/spi/spi_context.h

```diff
@@ -185,15 +185,13 @@ gpio_dt_flags_t spi_context_cs_active_level(struct spi_context *ctx)
 
 static inline void spi_context_cs_configure(struct spi_context *ctx)
```

Comment:
This function should return `gpio_pin_configure_dt` error code.

Reply:
Fixed.